packages/graphql-transformers-e2e-tests/src/__tests__/FunctionTransformerTestsV2.e2e.test.ts

```diff
@@ -0,0 +1,242 @@
+import { ResourceConstants } from 'graphql-transformer-common';
+import { GraphQLTransform } from '@aws-amplify/graphql-transformer-core';
+import { ModelTransformer } from '@aws-amplify/graphql-model-transformer';
+import { FunctionTransformer } from '@aws-amplify/graphql-function-transformer';
+import { AuthTransformer } from '@aws-amplify/graphql-auth-transformer';
+import { CloudFormationClient } from '../CloudFormationClient';
+import { Output } from 'aws-sdk/clients/cloudformation';
+import { GraphQLClient } from '../GraphQLClient';
+import { default as moment } from 'moment';
+import { cleanupStackAfterTest, deploy } from '../deployNestedStacks';
+import { S3Client } from '../S3Client';
+import { default as S3 } from 'aws-sdk/clients/s3';
+import { LambdaHelper } from '../LambdaHelper';
+import { IAMHelper } from '../IAMHelper';
+
+jest.setTimeout(2000000);
+
+const cf = new CloudFormationClient('us-west-2');
+const customS3Client = new S3Client('us-west-2');
+const awsS3Client = new S3({ region: 'us-west-2' });
+const BUILD_TIMESTAMP = moment().format('YYYYMMDDHHmmss');
+const STACK_NAME = `FunctionTransformerTests-${BUILD_TIMESTAMP}`;
```

Comment:
Can you add v2 in here and below as needed.

Reply:
I will update.

Reply:
Updated.